### ISSUE TITLE
PaddleX 2.1.0 图像分类验证集准确率偏低

### ISSUE THREAD

**Issue by Enn29** [PaddlePaddle/PaddleX#1320]
## Checklist:

1. 查找[历史相关issue](https://github.com/PaddlePaddle/PaddleX/issues)寻求解答
2. 翻阅[FAQ常见问题汇总和答疑](https://github.com/PaddlePaddle/PaddleX/blob/develop/docs/FAQ/FAQ.md)
3. 确认bug是否在新版本里还未修复
4. 如果bug是由PaddleX API 2.0导致，且该bug在develop分支里已修复，参考[FAQ Q4](https://github.com/PaddlePaddle/PaddleX/blob/develop/docs/FAQ/FAQ.md#gui%E7%9B%B8%E5%85%B3%E9%97%AE%E9%A2%98)替换内置PaddleX API

## 描述问题
GUI2.1.0版本进行图像分类训练时，验证集的准确率偏低，相同数据集相同模型参数设置相同，使用GUI 1.1.7版本训练时就没有这个问题
1.1.7版本准确率可以达到1, 2.1.0版本就只能达到0.6左右（这是使用样例训练出来的结果），使用自己的数据集验证集准确率都达不到0.6
参数设置：
![image](https://user-images.githubusercontent.com/63645468/147028718-c9c92260-2cef-4a32-9635-4f4fbb5ecaa2.png)

## 复现

1. 请提供您出现的报错信息及相关log（log的查找见 [FAQ Q2](https://github.com/PaddlePaddle/PaddleX/blob/develop/docs/FAQ/FAQ.md#gui%E7%9B%B8%E5%85%B3%E9%97%AE%E9%A2%98)）
![image](https://user-images.githubusercontent.com/63645468/147029088-c40e72fc-b74e-4c19-902b-70871d2d8fd5.png)

2. 请提供您使用的GUI版本号
GUI 2.1.0版本
3. 请提供您使用的操作系统信息，如Linux/Windows/MacOS
windows
4. 请问您使用的CUDA/cuDNN的版本号是？
![image](https://user-images.githubusercontent.com/63645468/147028604-9ecff0c6-3245-4fe5-8c52-ab7d9cc07e35.png)

**Comment by ucsk** [comment]
**Try training parameters:**
1. `Epoch`: 120
2. `Learning rate`: 0.0125
3. `Batch size`: 32
4. `Piecewise`: [30, 60, 90]

**Then open training enhancement strategy.**

**Comment by Enn29** [comment]
@ucsk 谢谢你的回复！
准确率有所提升，但训练时长也相应的延长
相较于1.1.7版本效果相差明显，2.1.0版本是在模型这块有优化么

**Comment by ucsk** [comment]
> @ucsk 谢谢你的回复！ 准确率有所提升，但训练时长也相应的延长 相较于1.1.7版本效果相差明显，2.1.0版本是在模型这块有优化么

我猜测，可能两次训练的数据集的划分方式不同，此外还有模型选择及其他参数均有所不同。最好是固定导入同样的数据集和划分文件，选择相同模型和超参数，之后再进行训练，验证新版本是否存在问题。

若考虑提升验证集上的指标，可以尝试将模型更换至更大型的模型，例如 `ResNet50_vd_ssld`、`ResNet101_vd_ssld`，调整Batch SIze，重新训练查看验证结果。

**Comment by Enn29** [comment]
好的
不过这次发现这个现象，是因为使用pplcnet进行训练效果不好，检查源码是发现有所优化，才会想着对比1.1.7版本看看
还有一个问题，restful里面新增的模型没有添加进去，这个麻烦你帮忙push一下

**Comment by Enn29** [comment]
@ucsk 你好，再次打扰一下，我使用**tutorials/train/image_classification/pplcnet.py**脚本，借用PaddleX GUI下划分的数据集，跑出来的效果非常好，验证集top1可以达到0.99，而使用GUI训练只能达到0.5左右。
按照pplcnet.py在GUI上设置参数，发现学习率不管怎样设置都是0.001
pplcnet.py参数没动，训练结果如下：
![image](https://user-images.githubusercontent.com/63645468/147200772-6b44a5a9-44c6-4045-b52b-e5a59fcbf2ec.png)
GUI参数设置如下（学习率在日志中显示为0.001）：
![image](https://user-images.githubusercontent.com/63645468/147200907-5da4aaf2-177e-42d4-807a-fac4055de535.png)
GUI训练log如下：
![image](https://user-images.githubusercontent.com/63645468/147201445-0c3b76f7-b563-45a9-8d53-891e12b0cb0b.png)
附件：
[out.zip](https://github.com/PaddlePaddle/PaddleX/files/7767337/out.zip)

**Comment by FlyingQianMM** [comment]
@Enn29 GUI的问题收到，我们会尽快定位原因。

您可以按照现有的使用方式，先继续用API开发模型。

**Comment by Enn29** [comment]
@FlyingQianMM restful上面的我找到在哪修改了，（paddlex_restful/restful/project/train/classification.py）
line73：
![image](https://user-images.githubusercontent.com/63645468/147458130-bdbcc7dc-419b-4234-a3ca-7e4607676810.png)
GUI上面的应该封装了，看不到

**Comment by Enn29** [comment]
@FlyingQianMM restful上面得我找到修改的位置了（paddlex_restful/restful/project/train/classification.py）
![image](https://user-images.githubusercontent.com/63645468/147458802-27ad80dd-2c68-4a94-bb56-f7014a303f4e.png)
GUI上面应该是封装了，暂时查不到

**Comment by FlyingQianMM** [comment]
@Enn29 感谢指正。restful的已经修复，见pr: https://github.com/PaddlePaddle/PaddleX/pull/1357 。GUI在下一个版本也会修复该问题。
